api/envoy/type/v3/http_status.proto

```diff
@@ -21,116 +21,172 @@ enum StatusCode {
   // `enum` type.
   Empty = 0;
 
+  // Continue - 100 status code.
```

Comment:
```suggestion
  // Continue - ``100`` status code.
```

@adisuissa i think these status codes should be literal - ie they are not counts but literally the code you receive

Reply:
Done for all status codes.